man/org.freedesktop.login1.xml

```diff
@@ -102,12 +102,19 @@ node /org/freedesktop/login1 {
                    in  b interactive);
       FlushDevices(in  b interactive);
       PowerOff(in  b interactive);
+      PowerOffWithInhibitors(in  b interactive);
```

Comment:
hmm, so first off, the `interactive` bool should go. the other methods have it because back in the day there was no field for that in the dbus header. There is now, hence the boolean doesn't have to be sent in the payload anymore.

But more importantly, i have the suspicion that sooner or later people want more flavours of this, i.e. maybe one that doesn't check inhibitors at all or so... Hence this hould have a uint64_t flags param i guess, which is how we do this these days... and then maybe rename this `PowerOffWithFlags` or so.

That way the new function becomes a true superset of the old function